examples/bert/bert_auto_opt_gpu.json

```diff
@@ -32,25 +24,68 @@
             }
         }
     },
+    "data_configs": {
+        "glue_mrpc": {
+            "name": "glue_mrpc",
+            "type": "HuggingfaceContainer",
+            "params_config": {
+                "model_name": "Intel/bert-base-uncased-mrpc",
```

Comment:
there is an automatic insertion of model_name and task from the input models so these are not necessary here. We wanted to make it easy for the user to just change the model name without having to update in multiple places.

Reply:
I will remove those in a follow up PR based on the outcome of this #1111. I will be required to revisit many of these examples on what we decided on the other one.

Reply:
Sure that sounds good to me!

Reply:
Done!